testing/proxytest/proxytest.go

```diff
@@ -143,6 +154,32 @@ func New(t *testing.T, optns ...Option) *Proxy {
 	return &p
 }
 
+func (p *Proxy) Start() {
+	p.Server.Start()
+	u, err := url.Parse(p.URL)
+	if err != nil {
+		panic(fmt.Sprintf("could parse fleet-server URL: %v", err))
+	}
+
+	p.Port = u.Port()
+	p.LocalhostURL = "http://localhost:" + p.Port
+
+	p.opts.logFn("running on %s -> %s", p.URL, p.LocalhostURL)
+}
+
+func (p *Proxy) StartTLS() {
+	p.Server.StartTLS()
+	u, err := url.Parse(p.URL)
+	if err != nil {
+		panic(fmt.Sprintf("could parse fleet-server URL: %v", err))
```

Comment:
Same here.

Reply:
Done in 979217c0b27b525e39d2344354e5e2e6363198b4